aten/src/ATen/native/SpectralOps.cpp

```diff
@@ -645,46 +645,30 @@ void _cufft_clear_plan_cache(int64_t device_index) {
   detail::getCUDAHooks().cuFFTClearPlanCache(device_index);
 }
 
-Tensor fft(const Tensor& self, const int64_t signal_ndim, const bool normalized) {
-  TORCH_WARN_ONCE(
-    "The function torch.fft is deprecated and will be removed in PyTorch 1.8. "
-    "Use the new torch.fft module functions, instead, by importing torch.fft "
-    "and calling torch.fft.fft or torch.fft.fftn.");
+static Tensor fft(const Tensor& self, const int64_t signal_ndim, const bool normalized) {
```

Comment:
One question here. Why we need `static`? Is that because we want internal linkage for these functions so they are visible in this file only? (fft, ifft, rfft, irfft).

Reply:
Yes, the only remaining users (`stft` and `istft`) are in the same file. Marked `static` so we know for sure it isn't in the public API, or even used elsewhere internally.